pkg/apierrors/apierrors.go

```diff
@@ -7,35 +7,35 @@ import (
 )
 
 var (
-	errCodeUnauthorizedUnknownUser                 = "ERR_UNAUTHORIZED_ACCESS_UNKNOWN_USER"
-	errCodeForbiddenInsufficientUserRole           = "ERR_FORBIDDEN_ACCESS_INSUFFICIENT_USER_ROLE"
-	errCodeForbiddenInsufficientProjectRole        = "ERR_FORBIDDEN_ACCESS_INSUFFICIENT_PROJECT_ROLE"
-	errCodeForbiddenUserNotProjectMember           = "ERR_FORBIDDEN_USER_NOT_PROJECT_MEMBER"
-	errCodeUserNotFound                            = "ERR_USER_NOT_FOUND"
-	errCodeProjectNotFound                         = "ERR_PROJECT_NOT_FOUND"
-	errCodeEnvironmentNotFound                     = "ERR_ENVIRONMENT_NOT_FOUND"
-	errCodeProjectUnprocessable                    = "ERR_PROJECT_UNPROCESSABLE"
-	errCodeEnvironmentUnprocessable                = "ERR_ENVIRONMENT_UNPROCESSABLE"
-	errCodeEnvironmentDuplicateName                = "ERR_ENVIRONMENT_DUPLICATE_NAME"
-	errCodeSettingsUnprocessable                   = "ERR_SETTINGS_UNPROCESSABLE"
-	errCodeProjectInvitationUnprocessable          = "ERR_PROJECT_INVITATION_UNPROCESSABLE"
-	errCodeProjectInvitationAlreadyExists          = "ERR_PROJECT_INVITATION_ALREADY_EXISTS"
-	errCodeProjectInvitationNotFound               = "ERR_PROJECT_INVITATION_NOT_FOUND"
-	errCodeProjectMemberAlreadyExists              = "ERR_PROJECT_MEMBER_ALREADY_EXISTS"
-	errCodeGithubIntegrationNotEnabled             = "ERR_GITHUB_INTEGRATION_NOT_ENABLED"
-	errCodeGithubClientUnauthorized                = "ERR_GITHUB_CLIENT_UNAUTHORIZED"
-	errCodeGithubClientForbidden                   = "ERR_GITHUB_CLIENT_FORBIDDEN"
-	errCodeGithubRepositoryNotConfiguredForProject = "ERR_GITHUB_REPOSITORY_NOT_CONFIGURED_FOR_PROJECT"
-	errCodeGithubRepositoryNotFound                = "ERR_GITHUB_REPOSITORY_NOT_FOUND"
-	errCodeGithubRepositoryInvalidURL              = "ERR_GITHUB_REPOSITORY_INVALID_URL"
-	errCodeProjectMemberNotFound                   = "ERR_PROJECT_MEMBER_NOT_FOUND"
-	errCodeProjectMemberUnprocessable              = "ERR_PROJECT_MEMBER_UNPROCESSABLE"
-	errCodeReleaseUnprocessable                    = "ERR_RELEASE_UNPROCESSABLE"
-	errCodeReleaseNotFound                         = "ERR_RELEASE_NOT_FOUND"
-	errCodeReleaseDuplicateTitle                   = "ERR_RELEASE_DUPLICATE_TITLE"
-	errCodeSlackIntegrationNotEnabled              = "ERR_SLACK_INTEGRATION_NOT_ENABLED"
-	errCodeGitTagNotFound                          = "ERR_GIT_TAG_NOT_FOUND"
-	errCodeGithubReleaseNotFound                   = "ERR_GITHUB_RELEASE_NOT_FOUND"
+	ErrCodeUnauthorizedUnknownUser                 = "ERR_UNAUTHORIZED_ACCESS_UNKNOWN_USER"
```

Comment:
`apierrors` error codes are now public, so `responseerrors` can translate the errors into appropriate response errors based on the code.